module/output/Overview/src/Overview.cpp

```diff
@@ -124,13 +124,6 @@ namespace module::output {
                             msg->ball_position_covariance = loc_ball->covariance.cast<float>();
                         }
                     }
-
-                    if (walk_path) {
```

Comment:
What's going on with this

Reply:
This block exists just to assign a message field which is never read. The message field was removed, and so this block was also removed.

Reply:
hmmmmm seems unrelated to this PR but sure